hack/python-sdk/swagger.json

```diff
@@ -62,7 +62,7 @@
         },
         "lastReconcileTime": {
           "description": "Represents last time when the job was reconciled. It is not guaranteed to be set in happens-before order across separate operations. It is represented in RFC3339 form and is in UTC.",
-          "$ref": "#/definitions/k8s.io.apimachinery.pkg.apis.meta.v1.Time"
+          "$ref": "#/definitions/v1.Time"
```

Comment:
So following snippet is the magic to change `k8s.io.apimachinery.pkg.apis.meta.v1.Time` to `v1.Time`?

```
	name = strings.Replace(name, "k8s.io/api/core/", "", -1)
 	name = strings.Replace(name, "k8s.io/apimachinery/pkg/apis/meta/", "", -1)
 	name = strings.Replace(name, "k8s.io/apimachinery/pkg/api/resource", "", -1)
```

Reply:
Yes, this one did the trick